android/build.gradle

```diff
@@ -63,32 +63,6 @@ dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
 
-    def exoPlayerVersion = safeExtGet("exoPlayerVersion", '2.11.4')
```

Comment:
On the top of this file there's more variables related to dash, his and smooth streaming that don't seem necessary anymore (are unused).